### PR TITLE
some more manifest stuff

### PR DESCRIPTION
**Pull request by andymckay**
* background, homepage_url and icons, yay!

### CODE REVIEW

File: tests/test.homepage_url.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+
+describe('/homepage_url', () => {
+
+  const validURLs = [
+    'https://example.com/some/page',
+    'http://foo.com',
+  ];
+
+  for (const validURL of validURLs) {
```
Comment (tofumatt) [inline tests/test.homepage_url.js]: `const` in loop again, can we use `let` instead? Thanks ^_^

File: tests/test.icons.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  it('should be valid', () => {
+    var manifest = cloneDeep(validManifest);
+    manifest.icons = {48: 'icon.png'};
+    validate(manifest);
+    assert.isNull(validate.errors);
+  });
+
+  it('should fail on integer value', () => {
+    var manifest = cloneDeep(validManifest);
+    manifest.icons = {48: 1};
```
Comment (tofumatt) [inline tests/test.icons.js]: Maybe overkill, but what about strings that are only numbers? Worth testing?

**Comment by tofumatt** [comment]
I wouldn't mind a test for multiple icons but I love adding extra tests.

r+wc

File: tests/test.homepage_url.js (hunk 1 above)
Comment (andymckay) [inline tests/test.homepage_url.js]: How about `var`? 🚽

File: tests/test.icons.js (hunk 2 above)
Comment (andymckay) [inline tests/test.icons.js]: Like `{48: '1'}`, that will pass because there's no restriction on the URL as long as its relative.

**Pull request merged**